baxter_description/urdf/baxter.urdf

```diff
@@ -1,1558 +0,0 @@
-<?xml version="1.0" ?>
```

Comment:
In this commit I added ``convert_to_urdf_and_sdf.sh`` - what if we just document how to use that in the README instead of adding the urdf? Often users will modify the wrong URDF, or modify one and source the other. It has been very misleading to me in the past.